src/http/request_context.rs

```diff
@@ -132,8 +136,9 @@ impl RequestContext {
   }
 
   /// unique id for this request.
-  pub fn id(&self) -> u128 {
-    self.id
+  #[cfg(feature = "unique_id")]
+  pub fn id(&self) -> &uuid::Uuid {
+    &self.id
```

Comment:
uuid::Uuid implements Copy and fits into a single CPU register on amd64. returning a reference is not necesarry. just return self.id directly and let auto copy do it.